config/default_config.yml

```diff
@@ -0,0 +1,164 @@
+# Default Configuration file for NyaaPantsu
+# PLEASE, do NOT change default_config.yml, create your own config.yml
+
+# Host of server
+host: localhost
+# port of server
+port: 9999
+# database type
+db_type: sqlite3
+# DBParams will be directly passed to Gorm, and its internal structure depends on the dialect for each db type
+db_params: ./nyaa.db?cache_size=50
+# logmode for database
+db_logmode: default
+# Environment should be one of: DEVELOPMENT, TEST, PRODUCTION
+environment: DEVELOPMENT
+# WebAddress : url of the website
+web_address: nyaa.pantsu.cat
```

Comment:
Maybe use *localhost* (without HTTPS) as a default website value and set the actual website in config.yml during deployment?

Otherwise every developer would have to be careful not to add their *config.yml* changes to their commits.